skaffold.yaml

```diff
@@ -45,7 +45,17 @@ deploy:
   statusCheckDeadlineSeconds: 120
   kubectl:
     manifests:
-    - ./kubernetes-manifests/*.yaml
+    - ./kubernetes-manifests/adservice.yaml
+    - ./kubernetes-manifests/cartservice.yaml
+    - ./kubernetes-manifests/checkoutservice.yaml
+    - ./kubernetes-manifests/currencyservice.yaml
+    - ./kubernetes-manifests/emailservice.yaml
+    - ./kubernetes-manifests/frontend.yaml
+    - ./kubernetes-manifests/paymentservice.yaml
+    - ./kubernetes-manifests/productcatalogservice.yaml
+    - ./kubernetes-manifests/recommendationservice.yaml
+    - ./kubernetes-manifests/redis.yaml
+    - ./kubernetes-manifests/shippingservice.yaml
```

Comment:
as you move out the loadgen manifest from kubernetes-manifests folder, please keep this explicit yaml list change here